src/routes/(docs)/Card/ExampleCard.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+	import { Card, Button, Typography } from '$lib';
+</script>
+
+<Card>
+	<div
+		class="relative h-80 rounded-xl bg-cover bg-center"
+		style="height: 200px; background-image: url('/jumbotron.jpg');"
```

Comment:
I would argue this should be an `<img />` component for a11y reasons if nothing else (allows setting alt text)